code/game/objects/items/stunbaton.dm

```diff
@@ -233,10 +233,19 @@
 			return FALSE
 	/// After a target is hit, we do a chunk of stamina damage, along with other effects.
 	/// After a period of time, we then check to see what stun duration we give.
-	L.Jitter(20)
-	L.set_confusion(max(confusion_amt, L.get_confusion()))
-	L.stuttering = max(8, L.stuttering)
-	L.apply_damage(stamina_loss_amt, STAMINA, BODY_ZONE_CHEST)
+	var/trait_check = HAS_TRAIT(target, TRAIT_STUNRESISTANCE)
```

Comment:
how did you manage to clone this locally, make all these changes, and then forget to try to compile it?